src/lib/types/process.d.ts

```diff
@@ -24,6 +24,16 @@ declare namespace Process {
     op?: any
   }
 
+  export type CLIArgs = {
```

Comment:
where is `CLIArgs` used?

Reply:
no anymore, becasue it's an even and a callback it's a bit more complicated than just that. Thanks for flagging.